ejabberd_bridge/management/commands/ejabberd_auth.py

```diff
@@ -78,6 +81,10 @@ def isuser(self, user_id=None, server="localhost"):
         self.logger.debug("Validating %s on server %s" % (user_id, server))
 
         try:
+            user_id = int(user_id)
```

Comment:
@fichie23 this should catch ValueError and return False instead of forward the exception